web/src/lib/components/Navbar.svelte

```diff
@@ -46,13 +46,51 @@
         }
     }
 
+    function closeMenu() {
+        menuDisplay = "hidden";
+    }
+
+    function openMenu() {
+        menuDisplay = "block";
+    }
+
+    function toggleMenu() {
+        if ( menuDisplay === "block") {
+            closeMenu();
+        } else if ( menuDisplay === "hidden") {
+            openMenu();
+        }
+    }
+
     function toggleTheme() {
         let html = <HTMLHtmlElement>document.querySelector('html');
-        let toggle = <HTMLInputElement>document.getElementById('theme-toggle');
-        html.dataset.theme = toggle.checked ? 'night' : 'light';
+        console.log(html.dataset.theme );
+        if ( html.dataset.theme === 'night') {
+            setLightTheme(html);
+        } else {
+            setDarkTheme(html);
+        }
+    }
+
+    function setDarkTheme(html: HTMLHtmlElement) {
+        html.dataset.theme = 'night';
+        themeToggle = 'Switch to Light Theme';
+    }
+
+    function setLightTheme(html: HTMLHtmlElement) {
+        html.dataset.theme = 'light';
+        themeToggle = 'Switch to Night Theme';
```

Comment:
The antonym of "light" is "dark", not "night"...

Reply:
took that from here:
```  daisyui: {
    themes: ['light', 'night'],
    darkTheme: "night"
  },```